integration/default_test.go

```diff
@@ -98,4 +98,59 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 			}).Should(ContainSubstring("AspNetCore.dll exists"))
 		})
 	})
+
+	context("image is built with BP_DOTNET_FRAMEWORK_VERSION set", func() {
+		var (
+			image  occam.Image
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("installs the version from $BP_DOTNET_FRAMEWORK_VERSION", func() {
```

Comment:
Technically, this test is asserting that the version from `RUNTIME_VERSION` is set. Can you use a `plan.toml` to decouple from the dotnet-core-runtime buildpack for this test? Ideally, I'd want to see version source `BP_DOTNET_FRAMEWORK_VERSION` selected as the version source.

Reply:
Yeah, it's unfortunate. We tried this but since we need the build plan buildpack before (to replace Runtime) and after (to require ASPNet), we would need 2 `plan.toml` files, and this isn't possible. 

It's a bit of a funny test case because yes it's using `RUNTIME_VERSION`, but `RUNTIME_VERSION` gets populated in the Runtime buildpack with whatever we set `BP_DOTNET_FRAMEWORK_VERSION`. So, in a way we're *kiiiind of* testing `BP_DOTNET_FRAMEWORK_VERSION`, but the buildpacks are just super tightly coupled.

TL;DR is that I'm not sure that there's a way to fix this.

Reply:
Ah, now that you wrote that out, I'm definitely remembering hearing either you or @arjun024 saying that exact thing out loud. My bad. This is because the ASPNet buildpack `requires` the dotnet runtime buildpack – totally makes sense. In which case the `it` is more like `it("considers BP_DOTNET_FRAMEWORK_VERSION as a version source")`. Given we're already testing that the buildpack installs the `RUNTIME_VERSION` version when it's present, I think this is sufficient.